File: ueueo-framework/ueueo-blob-storing/src/main/java/Abp/BlobStoring/BlobProviderDeleteArgs.cs
using System.Threading;
using JetBrains.Annotations;

namespace Volo.Abp.BlobStoring;

public class BlobProviderDeleteArgs : BlobProviderArgs
{
    public BlobProviderDeleteArgs(
        @Nonnull String containerName,
        @Nonnull BlobContainerConfiguration configuration,
        @Nonnull String blobName,
        CancellationToken cancellationToken = default)
        : base(
            containerName,
            configuration,
            blobName,
            cancellationToken)
    {
    }
}

File: ueueo-framework/ueueo-blob-storing/src/main/java/com/ueueo/blobstoring/BlobProviderDeleteArgs.java
package com.ueueo.blobstoring;

import com.ueueo.threading.CancellationToken;
import org.springframework.lang.NonNull;

public class BlobProviderDeleteArgs extends BlobProviderArgs {
    public BlobProviderDeleteArgs(
            @NonNull String containerName,
            @NonNull BlobContainerConfiguration configuration,
            @NonNull String blobName,
            CancellationToken cancellationToken) {
        super(
                containerName,
                configuration,
                blobName,
                cancellationToken);
    }
}
